packages/website/src/features/citation/ui/citation-data-filter.test.tsx

```diff
@@ -0,0 +1,71 @@
+import "@testing-library/jest-dom";
+import { render, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import CitationDataFilter from "./citation-data-filter";
+
+describe("Citation-data-filter rendering", () => {
```

Comment:
nit. If we wanted to test the rendering and behavior of the same component separately, we can do it like this:

```
describe("Component", () => {
  describe("rendering", () => {
    it("renders citation data...", () => { ... };
  });

  describe("interactions", () => {
    it("should...", () => { ... };
  });
});
```